aspnetcore/signalr/configuration.md

```diff
@@ -325,6 +325,66 @@ var connection = new signalR.HubConnectionBuilder()
 
 ---
 
+### Configure stateful reconnect
+
+SignalR stateful reconnect reduces the perceived downtime of clients that have a temporary disconnect in their network connection, such as when switching network connections or a short temporary loss in access.
+
+Stateful reconnect achieves this by:
+
+* Temporarily buffering data on the server and client.
+* Acknowledging messages received (ACK-ing) by both the server and client.
+* Recognizing when a connection is returning and replaying messages that might have been sent while the connection was down.
```

Comment:
Just an idea -- "up" seems a better counterpart to "down" than "returning". 

```suggestion
* Recognizing when a connection is up and replaying messages that might have been sent while the connection was down.
```

Reply:
We would be assuming a connection is "up" if it is both receiving and returning, so maybe we don't need to be explicit about returning.  Good point. I'll go with "up".